timeplus/client.go

```diff
@@ -283,44 +288,47 @@ func (s *TimeplusClient) queryStreamV2(sql string, batchCount int, batchBufferTi
 	config := utils.NewDefaultHTTPClientConfig()
 	res, err := utils.SSEHttpRequestWithAPIKey(http.MethodPost, createQueryUrl, query, config, s.apikey)
 	if err != nil {
-		return nil, nil, nil, fmt.Errorf("failed to create query : %w", err)
+		res.Body.Close()
+		return nil, fmt.Errorf("failed to create query : %w", err)
 	}
 
 	reader := bufio.NewReader(res.Body)
 	ch := make(chan rxgo.Item)
-	var header QueryInfo
+	var queryMetadata QueryInfo
 
-	// Read the header here
-	for {
-		line, err := readCompleteLine(reader)
+	// Read the first result from sse which should be an event
+	line, err := readCompleteLine(reader)
```

Comment:
why it is called `readCompleteLine` instead of just `readLine`? do we have another `readPartialLine`?